### DISCUSSION TITLE
switch_d is pulldown, update the doc ?

### DISCUSSION THREAD

**Discussion by ouinouin** [arendst/Tasmota#18402] [Q&A]
hi, while searching about switch_n and switch_d , we find some things in the discussions, but there should be a beginning of answer in the docs ,  in the components section most likely ,  : https://tasmota.github.io/docs/Components/#tasmota
and maybe also something in the buttons and switches pages ?

https://tasmota.github.io/docs/Buttons-and-Switches/

in order to be able to identify all the letters for the variant of each function.

**Comment by barbudor** [comment]
You are welcome to click on the pencil icon to submit a change in the doc
Thanks for your contribution

**Comment by ouinouin** [comment]
@barbudor  , i ll do this one effectively , i see that its more easy to contribute doc that i was thinking for a non coder.

do you validate my findings and assumptions that the _d stands for internal pull-down resistor ?

**Reply by barbudor** [reply]
Absolutely.
Available only on ESP32

**Comment by ouinouin** [comment]
i did update the table https://github.com/tasmota/docs/pull/1215 
i ll try to find time to make a paragraph about this on the relays and switches pages.
